teams/migration/membership.json

```diff
@@ -29,6 +29,7 @@
         "lonng",
         "overvenus",
         "suzaku",
+        "Tammyxia",
```

Comment:
/hold

In alphabetical order, this line should stay between "Little-Wallace" and "WangXiangUSTC".